FreeBSD's `install' doesn't support `-D'.  Thus, use GNU install instead,
since `stdenv-native' assumes that `ginstall' is available.

--- lcov-1.9/bin/install.sh	2012-03-02 16:53:28.000000000 +0100
+++ lcov-1.9/bin/install.sh	2010-08-06 13:05:17.000000000 +0200
@@ -34,7 +34,7 @@ do_install()
   local TARGET=$2
   local PARAMS=$3
 
-  ginstall -p -D $PARAMS $SOURCE $TARGET
+  install -p -D $PARAMS $SOURCE $TARGET
 }
 
